src/Versioned.php

```diff
@@ -474,6 +474,12 @@ protected function augmentSQL(SQLSelect $query, ?DataQuery $dataQuery = null)
             case 'archive':
                 $this->augmentSQLVersionedArchive($query, $dataQuery);
                 break;
+            case 'archive_only':
+                $this->augmentSQLVersionedArchiveOnly($query, $dataQuery);
+                break;
+            case 'removed_on_draft':
+                $this->augmentSQLVersionedRemovedOnDraft($query, $dataQuery);
```

Comment:
```suggestion
            case 'removed_on_draft':
                $this->augmentSQLVersionedRemovedFromDraft($query, $dataQuery);
```
Sorry just realised these are newly added.  Need to change the 'on' to 'from'

Reply:
Oh. I thought you have omitted that last time so that these protected methods could match the string they're using.... will update

Reply:
Done. Please tell me this is the last cycle of name changes.